packages/astro/src/layouts/BaseLayout.astro

```diff
@@ -1,17 +1,21 @@
 ---
 import Page from "@/layouts/Page.astro";
 import HeaderNav from "../components/HeaderNav.astro";
+import FooterNav from "../components/FooterNav.astro";
 
 interface Props {
-	title: string;
+  title: string;
 }
 
 const { title } = Astro.props;
 ---
 
 <Page title={title}>
-	<HeaderNav />
-	<main class="container">
-		<slot />
-	</main>
+  <HeaderNav />
+  <main class="container">
+    <slot />
+  </main>
+  <footer>
+    <FooterNav />
+  </footer>
```

Comment:
I think it would probably be better to include the <footer> wrapper in the component itself.  We should probably add a <header> to HeaderNav as well.